libbeat/common/config.go

```diff
@@ -39,12 +39,16 @@ type ConfigNamespace struct {
 	config *Config
 }
 
-var configOpts = []ucfg.Option{
+// Default opts used before reading the keystore configuration
+var defaultOpts = []ucfg.Option{
 	ucfg.PathSep("."),
 	ucfg.ResolveEnv,
 	ucfg.VarExp,
 }
 
+// ConfigOpts configures the options used to unpack the configuration
+var ConfigOpts = defaultOpts
```

Comment:
Why do we need to export this? `common.Config` type fully wraps go-ucfg, such that no other component has to deal with passing the options.

Reply:
I've removed the export of this field.